DataScience/GpxWavLabeler.ipynb

```diff
@@ -0,0 +1,356 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "id": "892d62b2-103d-4a13-bf6b-1729ad0b4905",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Manage imports\n",
+    "from pathlib import Path\n",
+    "import GpxWavLabeler as gwl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "id": "7a165d91-d274-43ae-ad93-b90e2624ead2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Assign data home directory, collection filename, sampling filepath, and clip home directory\n",
+    "data_home = str(Path(\"~\").expanduser() / \"Data\" / \"AISonobuoy\")\n",
+    "collection_filename = \"collection.json\"\n",
```

Comment:
What are these json files?

Reply:
These JSON files do exactly what you asked in the comment above. I will add some additional documentation to make it clearer.